x-pack/plugins/observability/public/hooks/use_get_user_cases_permissions.test.ts

```diff
@@ -1,79 +0,0 @@
-/*
```

Comment:
I removed this file because it's mostly covered by the cases plugin tests here:

x-pack/plugins/cases/public/client/helpers/capabilities.test.ts